Gradle build - Execution failed for task ':checkstyleMain'.

##### **Overview of the issue**

Gradle build failing with:

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':checkstyleMain'.
> Could not resolve all files for configuration ':checkstyle'.
   > Could not find com.puppycrawl.tools:checkstyle:${checkstyleVersion}.
     Required by:
         project :
   > Could not find com.puppycrawl.tools:checkstyle:${checkstyleVersion}.
     Required by:
         project : > io.spring.nohttp:nohttp-checkstyle:0.0.5.RELEASE

##### **Motivation for or Use Case**

<!-- Explain why this is a bug for you -->

 Single quotes prevent gradle propertie "checkstyleVersion" expansion

<!-- For bug reports, an unambiguous set of steps to reproduce the error -->

 Generate the project and run the gradle build 

##### **Suggest a Fix**

Replace the single quotes for doubles in build.gradle

checkstyle {
    toolVersion "${checkstyleVersion}"
    configFile file("checkstyle.xml")
    checkstyleTest.enabled = false
}

##### **JHipster Version(s)**                                  
                                                                    
```                                                                 
-- generator-jhipster@7.0.0-beta.0                                 
                                                                    
```                                                                 
                                                                                                                                     
##### **JHipster configuration, a `.yo-rc.json` file generated in th
```                                                                
                                                                    
<details>                                                           
<summary>.yo-rc.json file</summary>                                 
<pre>                                                               
{                                                                   
  "generator-jhipster": {                                           
    "blueprints": [],                                               
    "otherModules": [],                                             
    "applicationType": "monolith",                                  
    "baseName": "******",                                       
    "jhipsterVersion": "7.0.0-beta.0",                              
    "skipClient": false,                                            
    "skipServer": false,                                            
    "skipUserManagement": false,                                    
    "skipCheckLengthOfIdentifier": false,                           
    "skipFakeData": false,                                          
    "jhiPrefix": "jhi",                                             
    "entitySuffix": "",                                             
    "dtoSuffix": "DTO",                                             
    "testFrameworks": [],                                           
    "pages": [],                                                    
    "creationTimestamp": 1608553760200,                             
    "promptValues": {                                               
      "packageName": "******",                          
      "nativeLanguage": "en"                                        
    },                                                              
    "serviceDiscoveryType": false,                                  
    "reactive": false,                                              
    "authenticationType": "jwt",                                    
    "packageName": "*****",                            
    "serverPort": "8080",                                           
    "cacheProvider": "ehcache",                                     
    "enableHibernateCache": false,                                  
    "databaseType": "mongodb",                                      
    "buildTool": "gradle",                                          
    "serverSideOptions": ["websocket:spring-websocket"],            
    "websocket": "spring-websocket",                                
    "searchEngine": false,                                          
    "messageBroker": false,                                         
    "enableSwaggerCodegen": false,                                  
    "clientFramework": "vue",                                       
    "withAdminUi": true,                                            
    "clientTheme": "pulse",                                         
    "clientThemeVariant": "primary",                                
    "enableTranslation": false,                                     
    "nativeLanguage": "en",                                         
    "packageFolder": "****",                          
    "jwtSecretKey": "YourJWTSecretKeyWasReplacedByThisMeaninglessTex
sons",                                                              
    "devDatabaseType": "mongodb",                                   
    "prodDatabaseType": "mongodb",                                  
    "clientPackageManager": "npm",                                  
    "languages": ["en", "fr"],                                      
    "lastLiquibaseTimestamp": 1608555798000,                        
    "entities": ["Jobs"]                                            
  }                                                                 
}                                                                   
                                                                    
</pre>                                                              
</details>                                                          
                                                                    
                                                                    
##### **JDL for the Entity configuration(s) `entityName.json` files 
                                                                    
<details>                                                           
<summary>JDL entity definitions</summary>                           
                                                                    
<pre>                                                               
entity Jobs {                                                       
  name String                                                       
}                                                                   
dto Jobs with mapstruct                                             
paginate Jobs with pagination                                       
service Jobs with serviceImpl                                       
readOnly Jobs                                                       
                                                                    
</pre>                                                              
</details>                                                          
                                                                                                                     
##### **Environment and Tools**                                     
                                                                    
openjdk version "14.0.1" 2020-04-14                                 
OpenJDK Runtime Environment (build 14.0.1+7)                        
OpenJDK 64-Bit Server VM (build 14.0.1+7, mixed mode, sharing)      
                                                                    
git version 2.29.2.windows.3                                        
                                                                    
node: v12.18.1                                                      
                                                                    
npm: 6.14.10                                                        
                                                                    
yeoman: 3.1.1                                                       
                                                                    
Docker version 19.03.8, build afacb8b                               
                                                                    
docker-compose version 1.26.0, build unknown                        
```

##### **Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**

<!--
If the error is during an entity creation or associated with a specific entity.
If you are using JDL, please share that configuration as well.
-->

##### **Browsers and Operating System**

<!-- What OS are you on? is this a problem with all browsers or only IE8? -->

-   [ X] Checking this box is mandatory (this is just to show you read everything)

<!-- Love JHipster? Please consider supporting our collective:
👉  https://opencollective.com/generator-jhipster/donate -->

--- a/generators/server/templates/build.gradle.ejs
+++ b/generators/server/templates/build.gradle.ejs
@@ -240,7 +240,7 @@ gitProperties {
 }
 
 checkstyle {
-    toolVersion '${checkstyleVersion}'
+    toolVersion "${checkstyleVersion}"
     configFile file("checkstyle.xml")
     checkstyleTest.enabled = false
 }

